recipes/timml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here
+{% set name = "timml" %}
+{% set version = "5.0.0" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
```

Comment:
Is there a tarball available? See:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos